### PR TITLE
Make sure the new AppContext APIs are not part of netstandard2.0

### LINKED ISSUE #11682: Change ref projects to .NET Core
These AppContext APIs can be in .NET Core but not .NET Standard yet. We will need to change these these ref projects to .NET Core. 
Check file src/System.Runtime/src/ApiCompatBaseline.net463.txt
https://github.com/dotnet/corefx/pull/11275/files#r78505041

### PR DESCRIPTION
**Pull request by AlexGhiondea**
Fixes #11682

### CODE REVIEW

**Comment by joperezr** [comment]
@dotnet-bot test  Innerloop Windows_NT Debug Build and Test please (bug in the build has been fixed)

File: src/System.Runtime/ref/System.Runtime.cs [hunk 1]
```diff
@@ -83,8 +83,10 @@ public static partial class AppContext
         public static string BaseDirectory { get { throw null; } }
         public static void SetSwitch(string switchName, bool isEnabled) { }
         public static bool TryGetSwitch(string switchName, out bool isEnabled) { throw null; }
+#if netstandard11
```
Comment (akoeplinger) [inline src/System.Runtime/ref/System.Runtime.cs]: should be netcoreapp11 not netstandard11

Comment (AlexGhiondea) [inline src/System.Runtime/ref/System.Runtime.cs]: @akoeplinger ah yes! Great catch! Thanks!

**Pull request merged**